b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeSwitchOrg.js

```diff
@@ -20,25 +20,37 @@ import config from "../../../config.json";
 import callSwitchOrg from "../../apiCall/settings/callSwitchOrg";
 import { commonDecode } from "../../util/apiUtil/commonDecode";
 
-function getSubOrgId(token) {
-    
-    try {
+/**
+ * 
+ * @param token 
```

Comment:
Add parameter descriptions. Check other applicable places as well.

Reply:
Add a function description as well

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/8b059f39358c5652ea99e473f4cf34e23ad95e0b